app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1224,9 +1224,6 @@ public void updateMarker(boolean isBookmarked, Place place, @Nullable fr.free.nr
      * since grey icon may lead the users to believe that it is disabled or prohibited contribution
      */
     private void hideAllMArkers() {
-        if(currentLocationMarker==null){
```

Comment:
Hmm, as far as I can see at first sight this won't cause an NPE since currentLocationMarker is only used in addCurrentLocationMarker and nullcheck is being made inside of this method, already.

Reply:
Hi @neslihanturan 
I don't see where NPE could occur.